docroot/themes/custom/uids_base/templates/layouts/page--paragraph/layout--page--paragraph.html.twig

```diff
@@ -0,0 +1,62 @@
+{#
+/**
+ * @file
+ * Default theme implementation to display a one column layout.
+ *
+ * Available variables:
+ * - content: The content for this layout.
+ * - attributes: HTML attributes for the layout <div>.
+ *
+ * @ingroup themeable
+ * @see https://www.drupal.org/docs/8/api/layout-api/how-to-register-layouts
+ * @see https://v4-alpha.getbootstrap.com/layout/grid/
+ */
+#}
+
+{% set sidebar = content.sidebar|render %}
+
+{%
+  set classes = [
+  'layout',
+  'layout--page--paragraph',
+  'layout__container',
+  settings.extra_classes,
+  sidebar ? 'uids-menu-enabled' : 'uids-menu-disabled',
```

Comment:
@bspeare I think we should change this class name to something less specific to the menu, since it is actually controlling whether the sidebar appears. Maybe something like `.has-sidebar` or `.content--has-sidebar`.